layouts/partials/share_icons.html

```diff
@@ -1,4 +1,4 @@
-{{- $pageurl := .Permalink }}
+{{- $pageurl := .Page.Permalink }}
```

Comment:
`.Page` is depreciated and has been removed in Hugo `v0.92.0` https://github.com/gohugoio/hugo/releases/tag/v0.92.0